features/step_definitions/wishlist_steps.rb

```diff
@@ -51,3 +51,14 @@
   course = Course.where(department: course_dept, course_number: course_number)
   expect(WishlistItem.where(course: course, user: @current_user).first.notify).to be false
 end
+
+Then(/^([A-Z]+) (\d+) is the first class in my wishlist$/) do |course_dept, course_number|
+  course = Course.where(department: course_dept, course_number: course_number)
+  row = page.all('tr')[1] # The table header will be the first tr
+  course_data = row.all('td').first.text
+  expected_dept = course_data[0...4]
+  expected_number = course_data[4...course_data.length]
+  expected_course = Course.where(department: expected_dept, course_number:
```

Comment:
This is me being a little nit-picky..can you align the key-value pairs in the hash, rather than just breaking when the line gets too long?

So:
```ruby
expected_course = Course.where(department: expected_dept,
                                                       course_number: expected_number)
```

I thought rubocop would catch that, but I guess not.